### ISSUE TITLE
FBX Specular Intensity Incorrectly Imported

### ISSUE THREAD

**Issue by sbilkoloft** [assimp/assimp#841]
Hi,

Seems like the FBX converter has a little bug in it. In FBXConverter.cpp:1839 we have:
        const float Shininess = PropertyGet<float>(props,"Shininess",ok);
        if(ok) {
            out_mat->AddProperty(&Shininess,1,AI_MATKEY_SHININESS_STRENGTH);
        }

```
    const float ShininessExponent = PropertyGet<float>(props,"ShininessExponent",ok);
    if(ok) {
        out_mat->AddProperty(&ShininessExponent,1,AI_MATKEY_SHININESS);
    }
```

In FBX files "Shininess" and "ShininessExponent" have the same value. Doing a simple grep in the sample FBX files included in the repo confirms this point. The correct key to use is "SpecularIntensity" (grep also confirms this) which is the value used to scale the specular color.

I tried changing the key "Shininess" to "SpecularIntensity" but that didn't work. Seems like it should be added to a list somewhere else as well but I couldn't find where. If you doubt this, read the comment on the top of that function ;).

**Comment by mesilliac** [comment]
I think i fixed this in #1704

**Comment by kimkulling** [comment]
OK, will be closed.
